src/middlewared/middlewared/plugins/failover.py

```diff
@@ -582,10 +582,6 @@ async def unlock(self, options):
     @private
     @accepts(
         Str('pool_name'),
-        Dict(
-            'unlock_zfs_datasets',
-            Bool('restart_services', default=True),
-        )
```

Comment:
Could this make API backward compatible and somehow break failover during upgrade?
Just thinking out loud.

Reply:
@william-gr 12.0 signature is exactly the same `async def unlock_zfs_datasets(self, job, pool_name):` so this exact change should not be a failover breaker during CORE to SCALE update. Wrt 21.08 to 21.10 upgrade: is someone already using SCALE failover in production? 😄

Reply:
Nope

Reply:
@william-gr do you want me to change something then?

Reply:
@themylogin No, I was just asking to make sure you thought of that case.